### DISCUSSION TITLE
Need assistance - Looking at One Form, Multiple Tabs

### DISCUSSION THREAD

**Discussion by rcmsoftwaredevelopment** [serenity-is/Serenity#7012] [Help]
Hi all, I'm looking at the multiple tabs, one form which I need to do a Proof of Concept before the company invests in the framework.  At the moment, I'm using the free Serenity framework and trying to get two dialogs onto one form like the demo https://demo.serenity.is/BasicSamples/OtherFormInTabOneBar.  However, when I look at the source code this is using tsx files and I'm having a had job trying to map these over to ts and even the format.  Could someone help and point me in the right direction please?

**Comment by rcmsoftwaredevelopment** [comment]
Have my original source code which I'm playing about with. Has two tables Table1 and Table2. I'm trying to get both dialogs into one form.  
[Uploading SereneTabs.zip…]()

**Reply by LSNicholls** [reply]
Hi, I don't think your upload has worked, I couldn't download your source code.  But I've done this a lot and would like to help.

If you upload an image that shows the arrangement you are looking for as a wireframe or other sketch, maybe I (or somebody else) could explain from there.

Also, I don't know yet what you are trying to show, but normally the two tables would be in some sort of relationship.  I will assume that is the case for your sample?  Specify in your sketch what that relationship would be, so people can give you the best advice for your situation.
